medusa/providers/torrent/rss/nyaa.py

```diff
@@ -143,5 +143,21 @@ def parse(self, data, mode):
 
         return items
 
+    def _get_season_search_strings(self, episode):
+        search_string = {
+            'Season': []
+        }
+        season_search_keywords = ('batch', 'complete', 'season')
```

Comment:
I know it's a little offtopic regarding the PR. But I had to make this change before, I could fix the manual search cache query.

Maybe for season packs, we should go with using these keywords. Then we could store them like how we currently do for the 'PROPER' keywords.

@duramato thoughts?

Reply:
Continuing this in: https://github.com/pymedusa/Medusa/pull/3369.
I'll limit this PR to the db query changes.